packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -1,9 +1,95 @@
-import React from 'react';
-import './my-profile.scss';
+import React        from 'react';
+import PropTypes    from 'prop-types';
 import { localize } from 'deriv-translations';
+import                   './my-profile.scss';
+
+// TODO: This is just mockup data. it will be removed after Adding BE API.
+const user_data = {
+    firstName              : 'johny',
+    lastName               : 'bravo',
+    securityDeposit        : 0,
+    buyTrades              : 23,
+    sellTrades             : 25,
+    pastMonthTrades        : 42,
+    pastMonthCompletionRate: 99,
+    averageReleaseTime     : 5,
+    averageReleaseUnit     : 'min',
+};
+
+function capitalize (string) {
+    if (!string) {
+        return '';
+    }
+    return string[0].toUpperCase() + string.slice(1);
+}
+
+const Cell = (props) => {
+    return (
+        <div className='my-profile__cell'>
+            <div className='my-profile__cell__data'>
+                <div className='my-profile__cell__data__main'>
+                    {props.value}
+                    <sub className="my-profile__cell__data__main__dimension" >
+                        {props.dimension}
+                    </sub>
+                </div>
+                <div className='my-profile__cell__data__sub'>
+                    <div className='my-profile__cell__data__sub__upper'>
+                        {props.upperSubValue}
+                    </div>
+                    <div className='my-profile__cell__data__sub__lower'>
```

Comment:
Please do not tack on `__element` to BEM classes. Either use separate classes for each `__element`, or use a description delimiter (`-`) like this: `my-profile__cell-data-sub`.

Reply:
done